README.md

```diff
@@ -67,6 +70,31 @@ $ yarn test # for tests using STABLE environment
 $ yarn test:beta # for tests using BETA environment
 ```
 
+### Environments
+
+We currently support 4 different environments values (which are passed as `VTEX_ENV`), and this sections explains each one.
```

Comment:
I think it would be cool to add `local` as well, even if it's more straightforward. wdyt?